include/unifex/let_done.hpp

```diff
@@ -227,7 +227,7 @@ class _op<Source, Done, Receiver>::type {
   , receiver_((Receiver2&&)dest)
   {
     unifex::activate_union_member_with(sourceOp_, [&] {
-        return unifex::connect((Source&&)source, source_receiver{this});
+        return unifex::connect(std::move(source), source_receiver{this});
```

Comment:
Somehow this has broken the MSVC build.  I wonder if `Source` is sometimes an lvalue reference.

Reply:
Yup; line 324 instantiates an operation with an lvalue ref `Source` if the _Sender_ argument is an lvalue ref.  This is a `std::forward`.

_However_, most of the operation state types that I've written myself have a templated constructor that takes a forwarding reference to the _Sender_, rather than templating the whole type on a forwarding reference.  It seems like there'd be less duplicated code in the resulting binary if we could follow that pattern.